docs/en/routing/introduction.md

```diff
@@ -45,10 +56,29 @@ const r = renderer(() => <App />);
 r.mount({ registry });
 ```
 
--   Add an `outlet` widget to show the text "Home" when the `home` route is visited. Outlets are widgets that display something when a route is matched. The application's `src/routes.ts` file associates a route to an outlet via the outlet's `id` property.
+-   Add a `Route` widget to show the text "Home" when the `home` route is visited. `Route` is a widget that displays something when the path for the route id is matched. The application's `src/routes.ts` file associates a route to an id via the `Route`'s `id` property.
 
 > src/App.tsx
 
+```tsx
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Route from '@dojo/framework/routing/Route';
+
+const factory = create();
+
+export default factory(function App() {
+	return (
+		<div>
+			<Route id="home" renderer={() => <div>Home</div>} />
+			<Route id="about" renderer={() => <div>About</div>} />
+			<Route id="profile" renderer={() => <div>Profile</div>} />
+		</div>
+	);
+});
+```
+
+or using outlets and the `Outlet` widget, check out the [`Outlet` documentation](/learn/routing/outlets) for more information;
```

Comment:
incomplete thought / start of sentence?

Reply:
@dylans I think this is supposed to be a colon for the example below